pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -128,6 +132,9 @@ public void calculateBrokerHostUsage() {
         lastCollection = System.currentTimeMillis();
         this.usage = usage;
         usage.setCpu(new ResourceUsage(cpuUsage, totalCpuLimit));
+
+        executorService.schedule(catchingAndLoggingThrowables(this::calculateBrokerHostUsage),
```

Comment:
If there is an exception in the above method, we'll miss calling this `schedule`, which would stop broker host usage calculations.

Reply:
Note also that this PR's implementation is essentially proposing that we move to run with a fixed delay.

Reply:
OK, I will  use scheduleAtFixedDelay